CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-### Unreleased
+### 2.1.0 2017-07-25
```

Comment:
For some reason when I do 'git rebase master' the commits that this branch was behind on, compared to master, are shown as part of this pull request.